bcipy/task/paradigm/rsvp/copy_phrase.py

```diff
@@ -134,7 +134,7 @@ def __init__(self, win, daq, parameters, file_save, signal_model,
         self.parameters.add_entry(
             'preview_only',
             {
-                'value': True if self.parameters['preview_inquiry_progress_method'] == 0 else False,
+                'value': 'true' if self.parameters['preview_inquiry_progress_method'] == 0 else 'false',
```

Comment:
The Parameters module expects boolean values to be a string of 'true' or 'false'. This was the simplest fix for now. We may also want the Parameters module to be more lenient and allow Python boolean values, but that will require more testing to ensure it doesn't break any other existing code.

Reply:
We could add a check in `Parameters.check_valid_entry` that asserts values are strings or booleans are "true/false" in the meantime? I'd be okay with either approach